src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Please, rename this file to 'App.tsx' as we write our project in Typescript.

Reply:
done

Reply:
done